loc/items/gem.js

```diff
@@ -2,76 +2,77 @@ export default [
   {
     name       : {en: 'Lucid' },
     description: {
-      en: 'Gem description'
+      en: 'The sharp green edges catch sparkling light in the sun, making a beautiful and deadly sight.'
     }
   },
 
   {
     name       : {en: 'Melant' },
     description: {
-      en: 'Gem description'
+      en: 'Rough disc-shaped gray stone with strange druidical engravings encasing it.'
     }
   },
 
   {
     name       : {en: 'Turim' },
     description: {
-      en: 'Gem description'
+      en: ''
     }
   },
 
   {
     name       : {en: 'Fundo' },
     description: {
-      en: 'Gem description'
+      en: 'Brown desaturated gem, emitting putrid scents as you wear it around your neck.'
     }
   },
 
   {
     name       : {en: 'Royal' },
     description: {
-      en: 'Gem description'
+      en: 'Grand red gem enclosed in abyss dark fissures, granting your foes a sense of jealousy.'
     }
   },
 
   {
     name       : {en: 'Amary' },
     description: {
-      en: 'Gem description'
+      en: 'Lustrous blue artifact, exposed wispy white veins running along its spherical surface.'
     }
   },
 
   {
     name       : {en: 'Purum' },
     description: {
-      en: 'Gem description'
+      en: ''
     }
   },
 
   {
     name       : {en: 'Tara' },
     description: {
-      en: 'Gem description'
+      en: 'The sharp green edges catch sparkling light in the sun, making a beautiful and deadly sight.'
```

Comment:
This is good, maybe use thesaurus.com to find some alternatives for "beautiful".